pallets/runtime/src/bridge.rs

```diff
@@ -270,6 +289,50 @@ decl_module! {
             Ok(())
         }
 
+        /// Change the bridge limits.
+        #[weight = SimpleDispatchInfo::FixedOperational(20_000)]
+        pub fn change_bridge_limit(origin, amount: T::Balance, duration: T::BlockNumber) -> DispatchResult {
+            let sender = ensure_signed(origin)?;
+            ensure!(sender == Self::admin(), Error::<T>::BadAdmin);
+            <BridgeLimit<T>>::put((amount.clone(), duration.clone()));
```

Comment:
I think if there are some pending transactions (& those are valid at that moment) and because of some reason we change the bridge limit then there are high chances that those txn will become invalid. 

I think we have to live with this or do we want to mitigate this?

Reply:
There is no way to prevent this without storing the full history of deposits and recalculating buckets with every tx.

That being said, I don't think we need to worry about it since the admin can use the force handle tx function to force push any tx affected by this.